conans/client/generators/cmake_find_package.py

```diff
@@ -76,7 +76,7 @@ class CMakeFindPackageGenerator(GeneratorComponentsMixin, Generator):
 
         if({{ pkg_name }}_FIND_COMPONENTS)
             foreach(_FIND_COMPONENT {{ '${'+pkg_name+'_FIND_COMPONENTS}' }})
-                list(FIND {{ pkg_name }}_COMPONENTS "{{ pkg_name }}::${_FIND_COMPONENT}" _index)
+                list(FIND {{ pkg_name }}_COMPONENTS "{{ pkg_namespace }}::${_FIND_COMPONENT}" _index)
```

Comment:
Is there any difference between ``pkg_namespace`` and ``namespace``. In contents, it seems the value is the same one? If yes, then maybe we want to call it the same in the templates?

Reply:
Yes, it's the same, I'll change it